presto-delta/pom.xml

```diff
@@ -0,0 +1,290 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.facebook.presto</groupId>
+        <artifactId>presto-root</artifactId>
+        <version>0.264-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-delta</artifactId>
+    <description>Presto - Delta Table Connector</description>
+    <packaging>presto-plugin</packaging>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <enforcer.skip>true</enforcer.skip>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.facebook.presto.delta</groupId>
+            <artifactId>presto-delta-driver</artifactId>
+            <classifier>shaded</classifier>
+            <version>${project.version}</version>
+            <exclusions>
+                <exclusion>
```

Comment:
Exclusion of this will cause IDE showing cannot resolve symbol error. It might be better to set the scope delta-standalone dependency as compile in driver module, and then remove this execlusion.